README.md

```diff
@@ -24,7 +24,9 @@ bower install planout
 
 PlanOut.js provides an implementation of all PlanOut features (including the
 experiment class, interpreter, and namespaces). The underlying randomization ops in the JavaScript implementation return different
-results for efficiency reasons. If you are using PlanOut cross-platform and want to enable compatibility mode then you can enable it by calling ```ExperimentSetup.toggleCompatibleHash(true)```.
+results for efficiency reasons. If you are using PlanOut cross-platform and want to enable compatibility mode then you can enable it by utilizing the ```planout_core_compatible.js``` distribution bundle instead of the default ```planout.js``` bundle.
```

Comment:
@rawls238 @marcneuwirth @geoffdaigle 

How do you guys feel about the updated documentation? Is this clear?